src/styles/_catalog.scss

```diff
@@ -4,19 +4,32 @@
 
   max-width: 944px;
   margin: 0 auto;
+  perspective: 400px;
 
   &__card {
     display: flex;
     justify-content: center;
     align-items: center;
+
     width: 200px;
     height: 300px;
+
+    transition-duration: 300ms;
+    transition-timing-function: ease;
+    transition-property: color, transform;
+
     border: 1px solid #ccc;
     border-radius: 5px;
+
     margin-bottom: 48px;
   }
 
   &__card:not(:nth-child(4n)) {
     margin-right: 48px;
   }
+
+  &__card:hover {
```

Comment:
[Checklist](https://github.com/mate-academy/layout_catalog_hovers/blob/master/checklist.md) 9th point

Reply:
sorry, I forget for this(